src/omotes_sdk/internal/common/broker_interface.py

```diff
@@ -113,6 +115,51 @@ def to_argument(self) -> AioPikaQueueTypeArguments:
         return result
 
 
+@dataclass()
+class QueueMessageTTLArguments():
+    """Construct additional time-to-live arguments when declaring a queue."""
+
+    queue_ttl: Optional[timedelta] = None
+    """Expires and deletes the queue after a period of inactivity.
+    The timedelta must be able to be casted into a positive integer."""
+    message_ttl: Optional[timedelta] = None
```

Comment:
Perhaps we can add the relevant RabbitMQ link here for posterity?

Reply:
Updated in 3691c6257cb2daa7130fb39e33bb8f64de43e7ab